<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MovieDialog</class>
    <widget class="QDialog" name="MovieDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>500</width>
                <height>500</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Добавление фильма</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QLabel" name="titleLabel">
                    <property name="font">
                        <font>
                            <bold>true</bold>
                        </font>
                    </property>
                    <property name="text">
                        <string>Добавление нового фильма</string>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QFormLayout" name="formLayout">
                    <item row="0" column="0">
                        <widget class="QLabel" name="titleFormLabel">
                            <property name="text">
                                <string>Название*:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="1">
                        <widget class="QLineEdit" name="titleEdit">
                            <property name="placeholderText">
                                <string>Название фильма</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="0">
                        <widget class="QLabel" name="yearLabel">
                            <property name="text">
                                <string>Год*:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="1">
                        <widget class="QSpinBox" name="yearSpin"/>
                    </item>
                    <item row="2" column="0">
                        <widget class="QLabel" name="genreLabel">
                            <property name="text">
                                <string>Жанр:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="1">
                        <widget class="QComboBox" name="genreCombo"/>
                    </item>
                    <item row="3" column="0">
                        <widget class="QLabel" name="directorLabel">
                            <property name="text">
                                <string>Режиссер:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="3" column="1">
                        <widget class="QLineEdit" name="directorEdit">
                            <property name="placeholderText">
                                <string>Режиссер</string>
                            </property>
                        </widget>
                    </item>
                    <item row="4" column="0">
                        <widget class="QLabel" name="ratingLabel">
                            <property name="text">
                                <string>Рейтинг:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="4" column="1">
                        <widget class="QDoubleSpinBox" name="ratingSpin"/>
                    </item>
                    <item row="5" column="0">
                        <widget class="QLabel" name="posterLabel">
                            <property name="text">
                                <string>Постер:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="5" column="1">
                        <layout class="QHBoxLayout" name="posterLayout">
                            <item>
                                <widget class="QLabel" name="posterImageLabel">
                                    <property name="minimumSize">
                                        <size>
                                            <width>100</width>
                                            <height>150</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>100</width>
                                            <height>150</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string>Постер</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="loadPosterBtn">
                                    <property name="text">
                                        <string>Загрузить постер</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item row="6" column="0">
                        <widget class="QLabel" name="descriptionLabel">
                            <property name="text">
                                <string>Описание:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="6" column="1">
                        <widget class="QTextEdit" name="descriptionEdit">
                            <property name="maximumHeight">
                                <number>80</number>
                            </property>
                            <property name="placeholderText">
                                <string>Описание фильма...</string>
                            </property>
                        </widget>
                    </item>
                    <item row="7" column="1">
                        <widget class="QCheckBox" name="watchedCheck">
                            <property name="text">
                                <string>Просмотрен</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="standardButtons">
                        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>